common/static/js/src/string_utils.js

```diff
@@ -45,4 +47,278 @@
         return _.template(text, {interpolate: /\{(.+?)\}/g})(values);
     };
     this.interpolate_text = interpolate_text;
+
+    edx.StringUtils = edx.StringUtils || (function() {
```

Comment:
@cahrens The primary reason for copying the code was to do something quickly for today's marathon, as it was blocking a number of potential updates. This seemed like the quickest approach to get both Studio and LMS to work (but then I had the realization of the cleaner approach that would work for Studio).

My concern with doing what you suggest is that it is asynchronous, and so any non-RequireJS files could load first and then error trying to access the namespace. In Studio, everything is loaded via RequireJS so we know that nothing will have been invoked before the common libraries have initialized. Having said that, now that the marathon is over there is less urgency so it would make sense to try to come up with a solid approach that doesn't require copying.